Formula/podman.rb

```diff
@@ -148,10 +148,18 @@ def caveats
         for rootless containers to work properly.
       EOS
     end
+    on_macos do
+      <<-EOS
+        In order to run containers locally, podman depends on a Linux kernel.
+        One can be started manually using `podman machine` from this package.
+        To start a podman VM automatically at login, also install the cask
+        "podman-desktop".
+      EOS
+    end
```

Comment:
I don't think those are Homebrew-specific. This information is already present in the documentation of podman, where it belongs.

```suggestion
```